test/features/dictionary_objects/values.js

```diff
@@ -1,4 +1,4 @@
-let e = {};
+const e = {};
```

Comment:
This object is being changed in the next line, so it looks like it should be `let` instead of `const`